cmd/fyne/package_test.go

```diff
@@ -16,3 +16,14 @@ func Test_calculateExeName(t *testing.T) {
 	nonModulesApp := calculateExeName("testdata", "linux")
 	assert.Equal(t, "testdata", nonModulesApp)
 }
+
+func Test_isMobile(t *testing.T) {
+	oslist := []string{"darwin", "linux", "windows", "android/arm", "android/386", "android", "ios"}
```

Comment:
this should probably have an upper 'L' as it's a new word

Reply:
Also a test case should probably test some unexpected results for error cases or possible crashes